Tests/FluentAssertions.Specs/Types/MethodInfoSelectorSpecs.cs

```diff
@@ -367,6 +396,16 @@ internal class TestClassForMethodSelectorWithNonInheritableAttributeDerived : Te
         public override void PublicVirtualVoidMethodWithAttribute() { }
     }
 
+    internal class TestClassForMethodSelectorWithAsyncAndNonAsyncMethod
+    {
+        public async void PublicVirtualVoidAsyncMethod()
+        {
+            await Task.Yield();
+        }
+
+        public void PublicVirtualVoidNotAsyncMethod() { }
+    }
```

Comment:
Sorry to nag you about this, but as `virtual` is not relevant for testing async vs non-async I'd like to get rid of it.
```cs
internal class TestClassForMethodSelectorWithAsyncAndNonAsyncMethod
{
    public async Task PublicAsyncMethod()
    {
        await Task.Yield();
    }

    public Task PublicNotAsyncMethod() => Task.Completed;
}
```

Reply:
Oh, in the method name! Sorry!